### PR TITLE
Invisible links issue solved

### PR DESCRIPTION
**Pull request by darshikaguptaa**
Issue #705 Solved

### CODE REVIEW

File: src/components/Footer.js [hunk 1]
```diff
@@ -8,6 +8,7 @@ const Footer = () => (
     <p className="text-hack-fg light-mode:text-hack-dark-title">
       Disclaimer: This site is fan made and not affiliated with{' '}
       <a
+      style={{color:"black"}}
```
Comment (jenkoian) [inline src/components/Footer.js:11]: We use tailwind for styling, can you change this to a className attribute instead? Also, ideally want to re-use the colours declared in the tailwind config file. The colours are a bit of a mess and need sorting out, but in the mean time, you can probably use one of the colours there. See the surrounding p tag here for an example of some text colours and also the light-mode modifier.

Comment (darshikaguptaa) [reply src/components/Footer.js:11]: Sure I'll try to fix it

**Comment by darshikaguptaa** [comment]
Hey @jenkoian  I've changed the code according to tailwind config file

**Comment by vimode** [comment]
The same fix can be applied to links on hover.


As of now the hover effect too blends in with the background.

**Comment by darshikaguptaa** [comment]
@jenkoian are there any changes to be made?

**Comment by darshikaguptaa** [comment]
@jenkoian I've added the hover modifier as well

**Comment by jenkoian** [comment]
Thanks @darshikaguptaa

**Pull request merged**